### PR TITLE
The query listing was not accessible due to the python script appendi…

### PR DESCRIPTION
**Pull request by RobZoneNet**
…ng the wrong item. It would then spit out anything in that tag numerous time even with empty tags. This fixes castleteam issue #214.

I can demo this if you need me to.

### CODE REVIEW

File: castle/cms/browser/utils.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def focal_image_tag(self, brain, scale=None, className='', imageClassName='',
                 'data-scales-info': json.dumps(sizes_info)
             })
 
-        return tostring(el)
```
Comment (vangheem) [inline castle/cms/browser/utils.py]: You're losing the focus point div wrapper here aren't you?

**Comment by vangheem** [comment]
How is this related to #214

File: castle/cms/browser/utils.py (hunk 1 above)
Comment (RobZoneNet) [reply castle/cms/browser/utils.py]: <li class="castle-grid-block-item portal-type-news-item">
    <p>
      **<a href="https://wildcard.castlecms.io/blogs/wildcard-castlecms-and-open-source">
        </a>**</p><div class="focuspoint " data-base-url="https://wildcard.castlecms.io/blogs/wildcard-castlecms-and-open-source/@@images/image/" data-h="1425" data-scale="mini" data-focus-y="-0.158492822967" data-focus-x="-0.496041666667" data-scales-info="{&quot;listing&quot;: {&quot;h&quot;: 11, &quot;w&quot;: 16}, &quot;mini&quot;: {&quot;h&quot;: 139, &quot;w&quot;: 200}, &quot;preview&quot;: {&quot;h&quot;: 278, &quot;w&quot;: 400}, &quot;thumb&quot;: {&quot;h&quot;: 89, &quot;w&quot;: 128}, &quot;large&quot;: {&quot;h&quot;: 534, &quot;w&quot;: 768}, &quot;tile&quot;: {&quot;h&quot;: 44, &quot;w&quot;: 64}, &quot;icon&quot;: {&quot;h&quot;: 22, &quot;w&quot;: 32}, &quot;high&quot;: {&quot;h&quot;: 974, &quot;w&quot;: 1400}}" data-w="2048"><a href="https://wildcard.castlecms.io/blogs/wildcard-castlecms-and-open-source">Wildcard, CastleCMS and Open Source</a></div>**<a href="https://wildcard.castlecms.io/blogs/wildcard-castlecms-and-open-source">
      </a>
    <p></p>**
    
    
    <h3 class="title">
      <a href="https://wildcard.castlecms.io/blogs/wildcard-castlecms-and-open-source">Wildcard, CastleCMS and Open Source</a>
    </h3>
    <p class="description">The open source value proposition</p>
  </li>

Is what is rendered on the one blog post.  This part of grid.pt
<p tal:condition="has_image">
      <a href="${url}">
        <div tal:replace="structure python: utils.focal_image_tag(item, 'preview')" />
      </a>

    </p>
is getting odd. Spitting out empty tags along with the proper div tag

Comment (RobZoneNet) [reply castle/cms/browser/utils.py]: Since it is putting in numerous empty <a> tags it isn't accessible

Comment (RobZoneNet) [reply castle/cms/browser/utils.py]: Also it is 214 in the in house tickets

Comment (vangheem) [reply castle/cms/browser/utils.py]: Here is the raw html output:

```
<li class="castle-grid-block-item portal-type-image">
   <p>
      <a href="http://localhost:8080/Castle/image-repository/screen-shot-2018-10-22-at-2-19-03-pm.png/view">
   <div class="focuspoint " data-h="534" data-scales-info='{"mini": {"h": 49, "w": 200}, "thumb": {"h": 31, "w": 128}, "large": {"h": 190, "w": 768}, "high": {"h": 347, "w": 1400}, "listing": {"h": 3, "w": 16}, "tile": {"h": 15, "w": 64}, "preview": {"h": 99, "w": 400}, "icon": {"h": 7, "w": 32}}' data-scale="mini" data-focus-x="0.0" data-focus-y="-0.0" data-w="2152" data-base-url="http://localhost:8080/Castle/image-repository/screen-shot-2018-10-22-at-2-19-03-pm.png/@@images/image/"><img src="http://localhost:8080/Castle/image-repository/screen-shot-2018-10-22-at-2-19-03-pm.png/@@images/image/mini" alt="fsdfds" class=""></div>
   </a>
   </p>
   <h3 class="title">
      <a href="http://localhost:8080/Castle/image-repository/screen-shot-2018-10-22-at-2-19-03-pm.png/view">fsdfds</a>
   </h3>
   <p class="description"></p>
</li>
```

It seems to me that you're looking at the rendered output. Maybe it's javascript that is screwing it up.

**Comment by vangheem** [comment]
@RobZoneNet Closing this as we found out it was a different problem!

**Pull request closed without merging**